pkg/rancher-desktop/main/commandServer/settingsValidator.ts

```diff
@@ -331,16 +331,7 @@ export default class SettingsValidator {
       if (mergedSettings.experimental.virtualMachine.mount.type === MountType.NINEP) {
         errors.push(
           `Setting ${ fqname } to "${ VMType.VZ }" requires that experimental.virtual-machine.mount.type is ` +
-          `"${ MountType.REVERSE_SSHFS }" or "${ MountType.VIRTIOFS }".`);
-
-        return false;
-      }
-    }
-    if (desiredValue === VMType.QEMU) {
```

Comment:
There's no need to take this out (because it'll just never be executed, and we'll need it later).

Reply:
I was thinking about this at first, but I think we will just revert this whole PR when we re-enable virtiofs, and then make any other required changes as a followup.